File: app/src/main/java/com/communityx/activity/GroupInfoActivity.java
package com.communityx.activity;

import android.os.Bundle;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import butterknife.BindView;
import butterknife.ButterKnife;
import com.communityx.R;
import com.communityx.adapters.GroupInfoAdapter;
import com.communityx.utils.Utils;

public class GroupInfoActivity extends BaseActivity {

    @BindView(R.id.image_tail_one)
    ImageView imageEdit;
    @BindView(R.id.image_tail_two)
    ImageView imageAdd;
    @BindView(R.id.recycler_view)
    RecyclerView recyclerView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_group_info);
        ButterKnife.bind(this);

        setGroupToolbar(this,"Charity Life", "Created by You, today at 7:15 pm");
        setUpRecyclerData();
    }

    private void setUpRecyclerData() {
        recyclerView.setLayoutManager(new LinearLayoutManager(this));
        recyclerView.setAdapter(new GroupInfoAdapter(this));
    }
}

File: app/src/main/java/com/communityx/activity/GroupInfoActivity.kt
package com.communityx.activity

import android.os.Bundle
import android.support.v7.widget.LinearLayoutManager
import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import butterknife.BindView
import butterknife.ButterKnife
import com.communityx.R
import com.communityx.adapters.GroupInfoAdapter
import com.communityx.utils.Utils
import kotlinx.android.synthetic.main.activity_group_info.*

class GroupInfoActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_group_info)
        ButterKnife.bind(this)

        setGroupToolbar(this, getString(R.string.string_charity_life), getString(R.string.string_created_by_you))
        setUpRecyclerData()
    }

    private fun setUpRecyclerData() {
        recycler_view!!.layoutManager = LinearLayoutManager(this)
        recycler_view!!.adapter = GroupInfoAdapter(this)
    }
}
